.github/workflows/ci.yml

```diff
@@ -29,3 +29,16 @@ jobs:
       - run: yarn build
       - run: yarn test:coverage --runInBand
       - run: yarn test:ember
+
+  compatibility:
+    name: ember-try
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - run: yarn install --frozen-lockfile
+      - run: yarn test:compatibility
```

Comment:
I think I'd prefer to use a `matrix` "strategy" here and have dedicated jobs for each of the ember-try scenarios

Reply:
Sounds good. I was on the fence about this.

Reply:
Done!